amarachukwu-agbo/hello-books
# #158555488 Fix reviews component

#### What does this PR do?
Add bug fix for the reviews component

#### Description of Task to be completed?
- Add proptype validation for reviews component
- Change review form to stateless component
- Add jsdoc comments to review component


#### How should this be manually tested?

Clone the repo and install application dependencies. Run command  `npm run start:server` and `npm run start:client`. Go to 127.0.0.1:8080 and login into the application. Add an empty review, you should get a notification error.

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

#158555488

#### Screenshots (if appropriate)

N/A

#### Questions:
N/A
